src/main/resources/application.yaml

```diff
@@ -162,6 +162,9 @@ notification:
     responseReceived:
       emailId: a64bce9a-9162-47ca-b3e7-cf5f85ca7bdc
       smsId: f5b61f94-0b2b-4e8e-9c25-56e9830df7d4
+    appealDormant:
+      emailId: 976bdb6c-8a86-48cf-9e0f-7989acaec0c2
+      smsId: 1aa60c8a-1b6f-4ee1-88ae-51c1cef0ea2b
```

Comment:
Is there not a template for oral `appealDormant` - it's switched on in the `NotificationEventType.java` class, is this correct?

Reply:
yeah, I raised that to Mr Grimble and we decided to turn off for oral:)